Source/dom/style/Text.js

```diff
@@ -175,7 +175,10 @@ export function defineTextStyleProperties(Style) {
         return undefined
       }
 
-      const raw = paragraphStyle.alignment()
+      // eslint-disable-next-line
+      let raw = MSTextAlignmentConverter.archiveNSTextAlignment(
```

Comment:
May be worth adding a comment explaining why this is necessary, given the method name doesn't particularly help 🙂